android/src/main/java/com/adyenreactnativesdk/component/dropin/AdyenCheckoutService.kt

```diff
@@ -30,12 +26,19 @@ open class AdyenCheckoutService : DropInService(), DropInModuleListener {
         paymentComponentState: PaymentComponentState<*>,
         paymentComponentJson: JSONObject
     ) {
+        if (paymentComponentState is CardComponentState &&
+            paymentComponentJson.getJSONObject(PAYMENT_DETAILS_KEY).isNull(BRAND_KEY)
+        ) {
+            val cardType = paymentComponentState.cardType?.txVariant
+            paymentComponentJson.getJSONObject(PAYMENT_DETAILS_KEY).putOpt(BRAND_KEY, cardType)
```

Comment:
Did you make sure this actually ends up updating the `paymentComponentJson` param? Pretty sure you tested it but just a sanity check, I'm afraid whether `getJSONObject` returns a copy or the JSON object itself by reference.